dataactbroker/fileRoutes.py

```diff
@@ -75,6 +75,29 @@ def finalize_submission(upload_id):
     def check_status(submission):
         return get_status(submission)
 
+    @app.route("/v1/gtas_window/", methods=["GET"])
+    def gtas_window():
+        sess = GlobalDB.db().session
+
+        curr_date = datetime.utcnow()
+
+        gtas_window = sess.query(GTASSubmissionWindow).filter(
+                                                GTASSubmissionWindow.start_date < curr_date,
+                                                GTASSubmissionWindow.end_date > curr_date).one_or_none()
+
+        message = 'GTAS Window is not open'
+        data = None
+        window_open = False
+        data = {}
+
+        if gtas_window is not None:
+            message = 'GTAS Window is open'
+            data = gtas_window
+            window_open = True
+            data = {'start_date': str(gtas_window.start_date), 'end_date': str(gtas_window.end_date)}
+
+        return JsonResponse.create(StatusCode.OK, {"message": message, "open": window_open, "data": data})
```

Comment:
Do you need to return a message at all? You only really need to determine if it's open or not. The banner shouldn't display if the window isn't open so you can just manually set the message. Additionally, you don't need to return "open" because you can either return `None` or data for "data" and just compare that instead. Data will always be `None` when "open" is `False`.